diff --git a/dist/index.d.ts b/dist/index.d.ts
index 60649d9bd41a1ee238ed4741480337b80e251feb..afb9cbba0ddfddbe252c6e6e602126e97ea38cb9 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -142,7 +142,7 @@ declare type Options = {
     external?: (string | RegExp)[];
     /**
      * Replace `process.env.NODE_ENV` with `production` or `development`
-     * `production` when the bundled is minified, `development` otherwise
+     * `development` when the bundle is watched, `production` otherwise
      */
     replaceNodeEnv?: boolean;
     /**
diff --git a/dist/index.js b/dist/index.js
index c91d44e367b5f794b9bfb31fba2f982ac8e3d7b0..a0f070986a4bb19b510dc950140e57c9e143b462 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1620,7 +1620,7 @@ async function runEsbuild(options, {
     ...options.env
   };
   if (options.replaceNodeEnv) {
-    env.NODE_ENV = options.minify || options.minifyWhitespace ? "production" : "development";
+    env.NODE_ENV = !options.watch || options.minify || options.minifyWhitespace ? "production" : "development";
   }
   logger.info(format, "Build start");
   const startTime = Date.now();