diff --git a/dist/index.d.ts b/dist/index.d.ts
index 67b120f7296a3bd7fa369cea83b2310b67256818..2279dbbb8ed03caac360c0b36b19e8a742d9edc2 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -142,7 +142,7 @@ declare type Options = {
     external?: (string | RegExp)[];
     /**
      * Replace `process.env.NODE_ENV` with `production` or `development`
-     * `production` when the bundled is minified, `development` otherwise
+     * `development` when the bundle is watched, `production` otherwise
      */
     replaceNodeEnv?: boolean;
     /**
diff --git a/dist/index.js b/dist/index.js
index bcbdb40818d0b2d184c93190da1afc0d516e0841..f044ebef4e3949617fba5c137e566f9595111180 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1598,7 +1598,7 @@ async function runEsbuild(options, {
     ...options.env
   };
   if (options.replaceNodeEnv) {
-    env.NODE_ENV = options.minify || options.minifyWhitespace ? "production" : "development";
+    env.NODE_ENV = options.watch ? "development" : "production";
   }
   logger.info(format, "Build start");
   const startTime = Date.now();