diff --git a/dist/node/chunks/dep-561c5231.js b/dist/node/chunks/dep-561c5231.js
index cd14c84008c9a11fd9ed00c70b9e140a5fbe4c03..6040b3538035e1f162711e063b047bb4d293723a 100644
--- a/dist/node/chunks/dep-561c5231.js
+++ b/dist/node/chunks/dep-561c5231.js
@@ -57153,7 +57153,8 @@ function spaFallbackMiddleware(root) {
                     }
                 }
             }
-        ]
+        ],
+        disableDotRule: true
     });
     // Keep the named function. The name is visible in debug logs via `DEBUG=connect:dispatcher ...`
     return function viteSpaFallbackMiddleware(req, res, next) {