diff --git a/dist/node/chunks/dep-51c4f80a.js b/dist/node/chunks/dep-51c4f80a.js
index 6f2d3d43fcedbb752963705bcf66272b21dc7445..e37b66bab082a864094c5d608d0c66b59de170e8 100644
--- a/dist/node/chunks/dep-51c4f80a.js
+++ b/dist/node/chunks/dep-51c4f80a.js
@@ -59953,7 +59953,8 @@ function htmlFallbackMiddleware(root, spaFallback) {
                     }
                 }
             }
-        ]
+        ],
+        disableDotRule: true
     });
     // Keep the named function. The name is visible in debug logs via `DEBUG=connect:dispatcher ...`
     return function viteHtmlFallbackMiddleware(req, res, next) {